examples/applications/llm_safety_wrapper/llm_safety_wrapper.py

```
@@ -0,0 +1,610 @@
+            # Generation parameters
+            'max_tokens': 1000,
+            'high_risk_max_tokens': 500,
+            'temperature': 0.7,
+            'high_risk_temperature': 0.3,
+            
+            # Performance
+            'max_overhead': 0.1,
+            'timeout': 30,
+            
+            # System configuration (required by bridge)
+            'checkpoint_path': '/tmp/llm_safety_checkpoints',
```

Ruff (0.14.10)

Probable insecure usage of temporary file or directory: "/tmp/llm_safety_checkpoints"

(S108)